libraries/botbuilder/src/botFrameworkAdapter.ts

```diff
@@ -1907,6 +1922,20 @@ export class BotFrameworkAdapter
         await this.startWebSocket(nodeWebSocket);
     }
 
+    /**
+     * @private
```

Comment:
Shouldn't need this, the JSDoc lint rule is only enforced on non-private methods.

Reply:
Done, also cleaned up a few additional instances of this in the BotFrameworkAdapter.